.vscode/launch.json

```diff
@@ -4,13 +4,32 @@
   // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
   "version": "0.2.0",
   "configurations": [
+    {
```

Comment:
Added configurations to help with running Jest unit tests (for react/javascript)